jupyterlab_git/git.py

```diff
@@ -60,6 +60,35 @@ def communicate(self):
             return response
 
 
+class GitRemote:
```

Comment:
Why is this a separate class?

Reply:
The idea is to structure the backend with a syntax close to git commands; in this case, `git.remote.`_action_. But I could also integrate it directly as a method of `Git` class if you think it is cleaner.

Reply:
I went for a direct method of the `Git` class.